webapi/src/main/scala/org/knora/webapi/responders/v1/ResourcesResponderV1.scala

```diff
@@ -492,13 +479,15 @@ class ResourcesResponderV1(
   private def getResourceInfoResponseV1(
     resourceIri: IRI,
     userProfile: UserADM
-  ): Future[ResourceInfoResponseV1] =
+  ): Task[ResourceInfoResponseV1] =
     for {
-      (userPermissions, resInfo) <- getResourceInfoV1(
-                                      resourceIri = resourceIri,
-                                      userProfile = userProfile,
-                                      queryOntology = true
-                                    )
+      t <- getResourceInfoV1(
+             resourceIri = resourceIri,
+             userProfile = userProfile,
+             queryOntology = true
+           )
+      userPermissions = t._1
+      resInfo         = t._2
```

Comment:
maybe that's the python dev in me... but I like tuple decomposition.

Reply:
I do like it to, unfortunately this is not possible anymore with ZIO. ZIO does not implement `withFilter` and hence the syntactic niceness compiles with `Future` but not with ZIO.